asset/api/Asset.Samples/SearchAllResources.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright(c) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START asset_quickstart_search_all_resources]
+
+using Google.Api.Gax;
+using Google.Cloud.Asset.V1;
+using System.Collections.Generic;
+using System.Linq;
+
+public class SearchAllResourcesSample
+{
+    public SearchAllResourcesResponse SearchAllResources(string scope, string query = "", string[] assetTypes = null, int pageSize = 0, string orderBy = "")
```

Comment:
Sorry for not mentioning this before:
If the default values for parameters are meaningful, it's fine they are there, but in this case, they are really not, so better just remove them, here and everywhere.